diff --git a/cmake_modules/FindSuperCollider3.cmake b/cmake_modules/FindSuperCollider3.cmake
index d1372cc..f37950e 100644
--- a/cmake_modules/FindSuperCollider3.cmake
+++ b/cmake_modules/FindSuperCollider3.cmake
@@ -13,7 +13,7 @@
 # Find the telltale header file
 #
 GET_FILENAME_COMPONENT(SOURCEPARENT "${CMAKE_CURRENT_SOURCE_DIR}" PATH)
-find_path(SC_PATH NAMES include/plugin_interface/SC_PlugIn.h
+find_path(SC_PATH NAMES plugin_interface/SC_PlugIn.h
 	PATHS "${SOURCEPARENT}"
 	PATH_SUFFIXES SuperCollider)
 
